compiler/commands.nim

```diff
@@ -788,6 +788,15 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
   of "expandmacro":
     expectArg(conf, switch, arg, pass, info)
     conf.macrosToExpand[arg] = "T"
+  of "useversion":
+    expectArg(conf, switch, arg, pass, info)
+    case arg
+    of "0.19":
+      conf.globalOptions.incl optNimV019
```

Comment:
nit: is `optNimV019` related to `nimv019` in config/nim.cfg? as i understand this only affects gensym handling, so maybe it should be in the flag name, eg `--genSymVersion`

Reply:
We are moving over to "trunk based" development, that's why. These transition switches will be collected under ``useVersion``.